src/TaggedText.ts

```diff
@@ -342,6 +344,10 @@ export default class TaggedText extends PIXI.Sprite {
     }
 
     this.text = text;
+    this._boundsLRUCache =
```

Comment:
It looks like since it defaults to being disabled since the default Options object would not have a defined `cachedCapacity`. If this change is worth using, we should define something by default so it will always be used (unless maybe the user disables it on purpose).
 
Is there any chance that you could get an error if the capacity is too small? If not, I don't see any reason not to make it the default.